v1.6/get_raw_data.py

```diff
@@ -84,8 +84,9 @@ def get_raw_data(dest_dir='data/',params={}):
 
 
     print("*** Start crawling news pages ***")
-    for i in range(last_index,page_num):
-        target_url = url + "?max_results=50&page=" + str(i)
+    for i in range(last_index - 1, page_num + 1):
+        print("*** Start crawling news pages", i, " ***")
+        target_url = url + '?where={"style":{"$nin":["campaign"]},"isAdvertised":false,"isAdult":false,"state":{"$nin":["invisible"]},"categories":{"$nin":["57fca2f5c9b7a70e004e6df9","57f37a92a89ee20d00cc4a83"]}}&sort=-publishedDate&max_results&page=' + str(i)
```

Comment:
params should be applied to the variable 'url' in line 82, otherwise the page_num will be different.